### ISSUE TITLE
KeyError: 'page'

### ISSUE THREAD

**Issue by thorade** [nschloe/betterbib#30]
The following refin.bib file with just one single entry fails:
```
@InProceedings{KraeuchiEtAl2015:1,
  author = {Kr\"auchi, Philipp and Schluck, Thomas and Sulzer, Matthias},
  title = {Modelling of low temperature heating networks with IDA-ICE},
  booktitle = {Proceedings of International Conference CISBAT},
  doi = {10.5075/epfl-cisbat2015-827-832},
  pages = {827--832},
  address = {Lausanne, Switzerland},
  month = sep,
  year = {2015}
}
```

with the following error:
```
Reading from: refin.bib
Saving to: refout.bib

|----------| 0/1   0% [elapsed: 00:00 left: ?, ? iters/sec]Traceback (most recent call last):
  File "C:\Users\mthorade\Documents\Python\WinPython-32bit-2.7.9.2\python-2.7.9\Scripts\betterbib", line 99, in <module>
    _main()
  File "C:\Users\mthorade\Documents\Python\WinPython-32bit-2.7.9.2\python-2.7.9\Scripts\betterbib", line 48, in _main
    result = source.find_unique(entry)
  File "C:\Users\mthorade\Documents\Python\WinPython-32bit-2.7.9.2\python-2.7.9\lib\site-packages\betterbib\crossref.py", line 189, in find_unique
    if result['page'] == d['pages']:
KeyError: 'page'
```

If I delete the line ` pages = {827--832},` the error disappears (but there is still no result, as this conference paper is not indexed by crossref).
Tested with WinPython 2.7.9 32bit on Windows 10 64bit.

**Comment by nschloe** [comment]
Fixed in 2.1.1.

**Comment by thorade** [comment]
Confirmed, thanks a lot.
